gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -122,6 +139,12 @@ fluid.defaults("gpii.flowManager.local", {
             },
             target: "{that lifecycleManager}.options.components"
         },
+        "userListeners.lifecycleManager": {
+            record: {
+                lifecycleManager: "{gpii.flowManager.local}.lifecycleManager"
```

Comment:
Slightly clearer is to distribute "{gpii.flowManager.local}.lifecycleManager" to "{that gpii.userListener}.options.components.lifecycleManager"